projects/make-a-vis/src/lib/data-view/table/table.component.ts

```diff
@@ -144,6 +163,28 @@ export class TableComponent implements AfterViewInit, OnChanges, OnDestroy {
   ngOnDestroy() {
     loForEach(this.subscriptions, sub => sub.unsubscribe());
     this.subscriptions = [];
+
+    if (this.dataSubscription) {
+      this.dataSubscription.unsubscribe();
+    }
+
+    if (this.paginatorSubscription) {
+      this.paginatorSubscription.unsubscribe();
+    }
+  }
+
+  /**
+  * Updates data
+  */
+  updateData(): void {
+    this.dataSubscription = this.data$.subscribe((data) => {
```

Comment:
Subscription leak... Need to unsubscribe from the previous data subscription (if it exists) before creating a new one.